content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -3,16 +3,18 @@ reviewers:
 - sig-cluster-lifecycle
 title: Upgrading kubeadm clusters
 content_template: templates/task
+min-kubernetes-server-version: 1.18
 ---
 
 {{% capture overview %}}
 
 This page explains how to upgrade a Kubernetes cluster created with kubeadm from version
-1.16.x to version 1.17.x, and from version 1.17.x to 1.17.y (where `y > x`).
+1.17.x to version 1.18.x, and from version 1.18.x to 1.18.y (where `y > x`).
```

Comment:
@neolit123 , I am wondering if this file requires an update for every new release?
In the future, could you possibly use one of the docs shortcodes to automate some of the release version changes?